src/include/cephfs/metrics/Types.h

```diff
@@ -53,7 +53,8 @@ inline std::ostream &operator<<(std::ostream &os, const ClientMetricType &type)
     os << "OPENED_INODES";
     break;
   default:
-    os << "Unknown metric type: " << type;
+    os << "(UNKNOWN:" << static_cast<std::underlying_type<ClientMetricType>::type>(type) << ")";
+    break;
```

Comment:
was the segfault due to infinite recursion ?

Reply:
Yeah, it is.

I think it has run out of the memories' up limitation of the process.